File: src/SextantTG/SextantTG.ActiveRecord/Blog.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace SextantTG.ActiveRecord
{
    /// <summary>
    /// 日志记录
    /// </summary>
    public class Blog : BaseAR
    {
        /// <summary>
        /// 日志ID
        /// </summary>
        public string BlogId { get; set; }

        /// <summary>
        /// 日志创建者ID
        /// </summary>
        public string UserId { get; set; }

        /// <summary>
        /// 日志相关的景点ID
        /// </summary>
        public string SightsId { get; set; }

        /// <summary>
        /// 日志相关的旅行
        /// </summary>
        public string TourId { get; set; }

        /// <summary>
        /// 日志相关的子旅行ID
        /// </summary>
        public string SubTourId { get; set; }

        /// <summary>
        /// 日志标题
        /// </summary>
        public string Title { get; set; }

        /// <summary>
        /// 日志内容
        /// </summary>
        public string Content { get; set; }

        /// <summary>
        /// 创建时间
        /// </summary>
        public DateTime? CreatingTime { get; set; }
    }
}

File: src/App/src/com/ss/stg/model/Blog.java
package com.ss.stg.model;

import java.io.Serializable;
import java.util.Date;

/**
 * 日志
 * 
 * @author ggxx
 * 
 */
public class Blog implements Serializable {

	/**
	 * 
	 */
	private static final long serialVersionUID = -4204555102247271619L;
	private String blogId;
	private String userId;
	private String sightsId;
	private String tourId;
	private String subTourId;
	private String title;
	private String content;
	private Date creatingTime;

	public Blog() {
	}

	public String getBlogId() {
		return blogId;
	}

	public void setBlogId(String blogId) {
		this.blogId = blogId;
	}

	public String getUserId() {
		return userId;
	}

	public void setUserId(String userId) {
		this.userId = userId;
	}

	public String getSightsId() {
		return sightsId;
	}

	public void setSightsId(String sightsId) {
		this.sightsId = sightsId;
	}

	public String getTourId() {
		return tourId;
	}

	public void setTourId(String tourId) {
		this.tourId = tourId;
	}

	public String getSubTourId() {
		return subTourId;
	}

	public void setSubTourId(String subTourId) {
		this.subTourId = subTourId;
	}

	public String getTitle() {
		return title;
	}

	public void setTitle(String title) {
		this.title = title;
	}

	public String getContent() {
		return content;
	}

	public void setContent(String content) {
		this.content = content;
	}

	public Date getCreatingTime() {
		return creatingTime;
	}

	public void setCreatingTime(Date creatingTime) {
		this.creatingTime = creatingTime;
	}
}
